.github/workflows/build-timestamped-master.yml

```
@@ -13,8 +13,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set up JDK 11
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
+          distribution: 'adopt'
           java-version: 11
```

Shall we add quotes to the version as well (as a practice?)
```suggestion
          java-version: '11'
```

---

fixed in [2f8da03](https://github.com/ballerina-platform/module-ballerinax-kafka/pull/226/commits/2f8da03be8015a6128374629eed3e7eca76e37be)